BUILD.md

```diff
@@ -22,11 +22,28 @@ To compile Cemu, a recent enough compiler and STL with C++20 support is required
 
 For ubuntu and most derivatives:
 
-1) `sudo apt install -y libgtk-3-dev libsecret-1-dev libgcrypt20-dev libsystemd-dev freeglut3-dev clang-12 nasm ninja-build`
-2) Run `git clone --recursive https://github.com/cemu-project/Cemu`
-3) `cd Cemu`
-4) `mkdir build && cd build`
-5) `cmake .. -DCMAKE_BUILD_TYPE=release -DCMAKE_C_COMPILER=/usr/bin/clang-12 -DCMAKE_CXX_COMPILER=/usr/bin/clang++-12 -G Ninja`
-6) `ninja`
 
-Build instructions for other distributions will be added in the future!
+`sudo apt install -y libgtk-3-dev libsecret-1-dev libgcrypt20-dev libsystemd-dev freeglut3-dev clang-12 nasm ninja-build`
```

Comment:
I think that `cmake` is required too :)

Reply:
It’s not

Reply:
How are you going to run `cmake` on step 4 if `cmake` is not installed?

Reply:
Because it comes with ubuntu … already and this pr isn’t about ubuntu title says its all.

Reply:
> Because it comes with ubuntu … already

No, it doesn't, `cmake` is an optional package.

 > this pr isn’t about ubuntu title says its all. 

Since you're already modifying the file you could make life simpler also for Ubuntu users, no? :)